snuba/web/db_query.py

```diff
@@ -348,6 +348,9 @@ def execute_query_with_readthrough_caching(
     query_settings["query_id"] = query_id
 
     span = Hub.current.scope.span
+    sentry_sdk.set_tag("query_id", query_id)
```

Comment:
Do we need to be able to search by query_in in Sentry? Since a single query to Snuba may result in more than one query to Clickhouse, there is potentially more than one query_id per transaction. So this tag might be set and reset more than once, and only the last value actually preserved and sent to Sentry. Since each Clickhouse query would be a separate span though, the query IDs stored in the span data below would be more accurate.

Reply:
Thanks for the feedback. I have removed the call to set the tag. Only span.set_data is called now.